circuits/circom/core/non-qv/processMessages.circom

```diff
@@ -48,17 +47,10 @@ include "../../trees/incrementalQuinaryTree.circom";
     var STATE_LEAF_TIMESTAMP_IDX = 3;
     var msgTreeZeroValue = 8370432830353022751713833565135785980866757267633941821328460903436894336785;
 
-    // nb. The usage of SHA-256 hash is necessary to save some gas costs at verification time
-    // at the cost of more constraints for the prover.
-    // Basically, some values from the contract are passed as private inputs and the hash as a public input.
-
-    // The SHA-256 hash of values provided by the contract.
-    signal input inputHash;
-    signal input packedVals;
     // Number of users that have completed the sign up.
-    signal numSignUps;
+    signal input numSignUps;
     // Number of options for this poll.
-    signal maxVoteOptions;
+    signal input maxVoteOptions;
```

Comment:
given maxVoteOptions is always 5 ** voteOptionTreeDepth we can probably get rid of it completely as a input and just use the calculated one inside the circuit (MESSAGE_TREE_ARITY ** voteOptionTreeDepth)